tests/phpunit/api/v3/OptionValueCHTest.php

```diff
@@ -192,4 +192,91 @@ public function testCreateOptionValueChange() {
     $this->assertEmpty($updatedMap['values']);
   }
 
+  public function testContributionOnOptionvalueCHValueChange() {
+    $chFund = $this->callAPISuccess('OptionValue', 'create', [
+      'label' => 'Test Created CH Fund 1',
+      'option_group_id' => 'ch_fund',
+      'value' => 'CH+99999',
+    ]);
+    $chFundMap = $this->callAPISuccess('OptionValueCH', 'get', ['value' => 'CH+99999']);
+    $this->assertEquals($this->unallocatedFund['id'], $chFundMap['financial_type_id']);
```

Comment:
@monishdeb undefined index financial_type_id (hint you used get not get single just FYI)